src/src_V2/components/loading-spinner/LoadingSpinner.jsx

```diff
@@ -0,0 +1,9 @@
+import styles from './LoadingSpinner.module.css';
+
+export default function LoadingSpinner() {
```

Comment:
I think we can use Mui's spinners if our designers do not impose on us to use a custom one.

Reply:
They agreed, will change accordingly.